resources/analysisTools/indelCallingWorkflow/confidenceAnnotation_Indels.py

```diff
@@ -536,6 +562,9 @@ def main(args):
     parser.add_argument("-H", "--addhead", dest="additional_header", nargs="+", default=[],
                         help="String with additional header line infer multiple times for multiple additional lines.")
     parser.add_argument("-p", "--pid", dest="pid", nargs="?", help="Patient ID (default NA).", default="NA")
+    parser.add_argument("--skip_order_check", dest="skipOrderCheck", 
```

Comment:
I find the sentence hard to understand (and I am probably closest to a user that did not read the code). Maybe (?)

"Force the column 10 (1-based indexing) to be interpreted as control genotype (CONTROL_COL) and column 10 as tumor genotype (TUMOR_COL). A check for whether the order of the CONTROL_COL and TUMOR_COL columns in the VCF matches the SM tags in BAM the file is skipped! "

Reply:
Thanks! Have updated this.